src/pages/Login/Login.js

```diff
@@ -0,0 +1,63 @@
+import { React, useState } from 'react';
+import './Login.scss';
+
+const Login = () => {
+  const [inputValue, setInputValue] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const { id, pw } = inputValue;
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInputValue({
+      ...inputValue,
+      [name]: value,
+    });
+  };
+
+  return (
+    <div>
+      <section className="login">
+        <div className="loginTitle">CROWS SEVEN</div>
+        <div className="inputArea">
+          <form className="idLine">
+            <span className="idInputLine">Id</span>
+            <input
+              className="idInput"
+              type="text"
+              name="id"
+              onChange={handleInput}
+            />
+          </form>
+          <form className="pwLine">
+            <span className="pwInputLine">Password</span>
+            <input
+              className="pwInput"
+              type="password"
+              name="pw"
+              onChange={handleInput}
+            />
+          </form>
+        </div>
+        <button
+          className="signBtn"
+          disabled={id.includes('@') && pw.length >= 5 ? false : true}
```

Comment:
```suggestion
          disabled={!(id.includes('@') && pw.length >= 5)}
```
최초에 리뷰 드렸던 부분 인 것 같은데 똑같은 실수로 다시 돌아가시면 안됩니다!!

Reply:
빨리 만들어서 기능 많이 치는게 중요한게 아니라 하나를 하더라도 더 많은 걸 얻어가는 것에 중점을 두고 진행해주세요